src/design-tokens/tier-2-usage/colors-icon.json

```diff
@@ -100,7 +100,7 @@
                 "value": "{eds.color.other.ruby.500}"
               },
               "hover": {
-                "value": "{eds.color.other.ruby.500}"
+                "value": "{eds.color.other.ruby.600}"
```

Comment:
Can you help me understand this line?

Reply:
oh yep -- this was supposed to be `600` all along per the [Figma](https://www.figma.com/file/IfMWAtWqNrQprImlco4pi0/EDS---UI-Foundations-(BETA)?node-id=0%3A259). think there was a typo bringing it in originally

Reply:
Awesome, thanks!